packages/docusaurus-mdx-loader/src/remark/contentTitle/__tests__/index.test.ts

```diff
@@ -19,21 +19,25 @@ async function process(
 describe('contentTitle remark plugin', () => {
   describe('extracts data.contentTitle', () => {
     it('extracts h1 heading', async () => {
-      const result = await process(`
+      const result = await process(
+        `
 # contentTitle 1
 
 ## Heading Two {#custom-heading-two}
 
 # contentTitle 2
 
 some **markdown** *content*
-  `);
+  `,
+        {removeContentTitle: true},
+      );
```

Comment:
> Some test needed removeContentTitle: true throw Cannot handle unknown node 'mdxJsxFlowElement' and test returns content unmodified still fails thrown: "Exceeded timeout of 15000 ms for a test.


Why would that test now need `removeContentTitle: true`. I tested locally and if it now timeouts, there must be a very good reason that we should find out.